test/auth_test.rb

```diff
@@ -457,6 +457,18 @@ def app
       assert_includes response_cookie, "heroku_session=1;"
       assert_includes response_cookie,
         "heroku_session_nonce=8bb579ed-e3a4-41ed-9c1c-719e96618f71;"
+
+      assert response_cookie =~ /^heroku_user_session=(.+)$/,
+             "it should contain heroku_user_session"
+      # current version of `fernet` barks about URL encoded string
+      cipher = CGI::unescape($1.split(";")[0])
+      coder = Identity::CookieCoder.new(
+        Identity::Config.heroku_root_domain_cookie_encryption_key)
+      payload = coder.decode(cipher)
+
+      assert_equal "06dcaabe-f7cd-473a-aa10-df54045ff69c", payload["user"]["id"]
+      assert_equal "email@heroku.com", payload["user"]["email"]
```

Comment:
Nope, it should get from [this](https://github.com/heroku/identity/blob/root_domain_cookie/test/service_stubs/heroku_api_stub.rb#L98-L102).